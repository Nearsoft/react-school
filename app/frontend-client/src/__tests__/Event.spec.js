import React from "react";
import TestRenderer from 'react-test-renderer';
import Event from '../components/event/index';

describe('Event Component', () => {
    it('Render the Event component and check for properties', () => {
        const event = {
            title: '',
            description: '',
            limit: 0,
            address: '',
            user_id: 1
        };

        const testRenderer = TestRenderer.create(
            <Event key={0} event={event} />
        );
        console.log(testRenderer.toJSON());

        expect(testRenderer.toJSON().type).toEqual('div');
        expect(testRenderer.toJSON().props.className).toEqual('event-card r-padding-large r-margin-bottom-small');
        expect(testRenderer).toMatchSnapshot();
    });
});