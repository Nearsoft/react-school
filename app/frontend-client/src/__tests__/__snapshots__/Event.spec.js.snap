// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Component Render the Event component and check for properties 1`] = `
<div
  className="event-card r-padding-large r-margin-bottom-small"
>
  <div>
    <h3 />
    <p>
      
    </p>
    <a
      className="btn btn-primary"
      href="#/events/undefined"
    >
      Inscribirse
    </a>
  </div>
</div>
`;
